### DISCUSSION TITLE
Where's the Expander Plugin?

### DISCUSSION THREAD

**Discussion by sethicis** [wwmm/easyeffects#1367] [Q&A]
I just installed the EasyEffects Flatpak package and was configuring some plugins for my mic when I noticed that I couldn't find the expander plugin.  It doesn't show up in the listing or search, but the README file says that it is part of the default plugins included.

Is it under a different name?  Am I overlooking it somehow?

**Comment by wwmm** [comment]
That was probably inserted in the readme by mistake. There is no expander plugin.

**Reply by wwmm** [reply]
I updated the readme removing the mention to an expander from it.
